ScoutSuite/providers/azure/rules/findings/securitycenter-standard-tier-not-enabled.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dashboard_name": "Security Center",
+    "description": "Standard tier not enabled",
+    "rationale": "You should enable Standard Tier pricing for your Security Center. See CIS 2.1.",
+    "path": "securitycenter.pricings.id",
+    "display_path": "securitycenter.pricings.id",
+    "conditions": [ "and",
+        ["securitycenter.pricings.id.pricing_tier", "notEqual", "Standard" ]
+    ],
+    "id_suffix": "standard_pricing_enabled"
```

Comment:
Your id suffixes don't seem to be used anywhere.

Reply:
Copy pasted! Should I remove them entirely?